pages/common/pngquant.md

```diff
@@ -0,0 +1,36 @@
+# pngquant
+
+> PNG converter and lossy image compressor.
+> More information: <https://pngquant.org/>.
+
+- Compress a PNG as much as possible and write result to a new file:
+
+`pngquant {{path/to/file.png}}`
+
+- Compress a PNG and override original:
+
+`pngquant --ext .png -f {{path/to/file.png}}`
+
+- Try to compress a PNG with custom quality (skip if below the min value):
+
+`pngquant -Q {{0-100}} {{path/to/file.png}}`
```

Comment:
```suggestion
`pngquant --quality {{0-100}} {{path/to/file.png}}`
```

Optional change: be consistent: use long options everywhere unless it's better to use the short ones because the long counterparts are easily memorable.